modules/browser_object_tabbar.py

```diff
@@ -29,6 +30,18 @@ def new_tab_by_button(self) -> BasePage:
             self.get_element("newtab-button").click()
         return self
 
+    def new_tab_by_keys(self, sys_platform) -> BasePage:
+        """Use keyboard shortcut to open a new tab"""
+        if sys_platform == "Darwin":
+            ActionChains(self.driver).key_down(Keys.COMMAND).send_keys("t").key_up(
```

Comment:
You have access to ActionChains through the `BasePage` model, so if you don't mind changing calls to `ActionChains` to `self.actions`, and not import ActionChains here, that would be great.